Add Uninstall docs

BenWhitehead opened the pull request.

BenWhitehead commented:
@ssk2 Please review. /cc @air

air commented:
I think this is a little cryptic:
`Zookeeper Node provided to CASSANDRA_ZK`
`Using zkCli run the rmr command`

1. how do I find that value CASSANDRA_ZK
2. where can I find zkCli, on what machine do I run it, what rmr args do I need.

This is great btw, will serve as reference example for others.

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+
+1. Run `dcos package uninstall cassandra`
+2. Zookeeper Node provided to `CASSANDRA_ZK`
+    * Using zkCli run the `rmr` command
```
What path do we need to `rmr`?

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+
+# Manual Uninstall on DCOS
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
```
`cleanup` -> `complete`

ssk2 replied:
`Uninstall` -> `uninstall`

ssk2 replied:
`items` -> `actions`

ssk2 replied:
Should we use "Service" instead of Framework here?

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+------------
+
+
+# Manual Uninstall on DCOS
```
Can we make `#Manual Uninstall` an additional H1 heading at the top and make this and the other heading H2 (i.e. `##`)?

ssk2 replied:
(The title Markdown properties simply sets the HTML `<title>`.

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+    
+# Manual Uninstall on Mesos
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
```
Same word changes as above.

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+
+1. Delete marathon app
+2. Shutdown framework
+    1. Get framework id from `http://mesos-master:5050/#/frameworks`
```
`id` - > `ID`

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+2. Zookeeper Node provided to `CASSANDRA_ZK`
+    * Using zkCli run the `rmr` command
+3. Remove the directory provided to `CASSANDRA_DATA_DIRECTORY`
+    * The default behavior is to write the Data into the Mesos Sandbox and does not need to be manually cleaned unless a specific value was provided.
```
`the Data` -> `data`

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+3. Zookeeper Node provided to `CASSANDRA_ZK`
+    * Using zkCli run the `rmr` command
+4. Remove the directory provided to `CASSANDRA_DATA_DIRECTORY`
+    * The default behavior is to write the Data into the Mesos Sandbox and does not need to be manually cleaned unless a specific value was provided.
```
`the Data` -> `data`

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+2. Shutdown framework
+    1. Get framework id from `http://mesos-master:5050/#/frameworks`
+    2. run `curl -d "frameworkId=<framework_id_from_step_1>" http://mesos-master:5050/master/shutdown`
+3. Zookeeper Node provided to `CASSANDRA_ZK`
```
`Zookeeper` -> `ZooKeeper`

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
+
+1. Run `dcos package uninstall cassandra`
+2. Zookeeper Node provided to `CASSANDRA_ZK`
```
`Zookeeper` -> `ZooKeeper`

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,35 @@
+# Manual Uninstall on DCOS
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
+
```
Worth pointing out that this will _completely_ uninstall the Cassandra-on-Mesos framework - data and all.

BenWhitehead commented:
@ssk2 I incorporated all your proposed edits and added some of the notes you mentioned. Please review.

@air In the case of DCOS I've added in the default values for the `rmr` command.  The point about how to find the values I think plays into "How do we expose configuration values for Services", WDYT?

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,42 @@
+
+## Manual Uninstall on DCOS
+
+_All instructions below assume the default values were used when installed on DCOS_
```
What's happening with the L in below here/

ssk2 commented on docs/docs/uninstall.md:
```diff
@@ -0,0 +1,42 @@
+
+To uninstall the Cassandra Mesos Framework complete the following actions:
+
+1. Delete marathon app
```
`marathon` -> `Marathon`

ssk2 commented:
One small correction, otherwise good!
